free-2.md

```diff
@@ -11,41 +11,58 @@ _What need is met by your idea? Who are you helping? What is that person's
+
+
 ### Why a language?
 
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A dsl that could be accessable during game time would allow for users to update statistics in real time. The need for real time statisic updates at the non profesional level is fairly high. It would be amazing if such a laguage could be used to provide accurate statistics for players coaches and familly members in real time. As well as get rid of the need to record the statisic and maunally input each stat into some interface later on. 
+
 ### Why you?
 
 _What excites you about this idea? How did you come up with it?_
 
+What excites me about this idea is its ability to start small lets say for the average atheletes dad recording player stats for their children, and also have the ability to grow to the level where college coaches who spend hours after a game inputing stats can do it instantly durring the game. I came up with this idea after trying to watch my brothers football games from out of state. My brother plays football in highschool in colorado, the only way I could watch his game was if they put it online. If the game wasnt online I still wanted a way to see how he did. I know my dad takes stats at every game and provides the stats to the coaches too uplaod to maxpreps which is a high school sports stats collection for all teams in the USA. But I can not call him durring the game and ask what are the team stats, and I have to wait a day for these stats to be upaloded onine. There has to be a better way and maybe a DSL could fix this problem.
+
 ### Domain
 
 _Describe the project's domain in five words._
 
+Language for real time sports statistics
+
 ### Interface (syntax)
 
 _How might the user interact with the language? What does programming look
 like? Why is this the right way to interact with the problem domain?_
 
+Someone such as a parent or coach could use their phone or computer and write a sentence such as "player 1 passed for 33 yards to player 2" the DSL would know to add 33 yards to player 1's passing total and 33 yards to player 2's catching yard total. I feel like this laguage would work best using a natural laguage syntax because it would allow for the typical user such as a parent or coach to use it without having to know a laguage. Another possible way to structure this language would be to use sports terminology such as TFL(player1, num yards) which would mean that player 1 had a tackle for a loss of num yards. For coaches and parents who understand this terminology this laguage would be very easy to infer. 
+
```

Comment:
I like that there are a few options for how to interact with the language, and both ways make sense to the domain users.